### ISSUE TITLE
Update about new features and updates about the history page

### ISSUE THREAD

**Issue by Hasthikagirihagama** [acf-sack/sack-site#315]
**Is your feature request related to a problem? Please describe.**
Update any new features needed for the history page of St.Anthony's College 
https://www.sack.edu.lk/history/

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Comment by Yoshitha-SACK** [comment]
Shall I work on this?

**Comment by akeel230** [comment]
Rev, Fr Henry has been informed regarding this issue already, I requested the prize-giving archive booklets from him.
He said he will be able to provide us with the Hard copies. @Hasthikagirihagama or the Media unit president can go and collect it from him.

**Comment by akeel230** [comment]
There are no softcopies, so we will have to extract the information from the provided hard copies.

**Comment by Piumal1999** [comment]
@akeel230 @Hasthikagirihagama any update on this?

**Comment by akeel230** [comment]
@Piumal1999 there was a message in the ACF telegram group saying they received it from Father.

**Comment by Hasthikagirihagama** [comment]
> @Piumal1999 there was a message in the ACF telegram group saying they received it from Father.

Yes, we received the hard copies from Rev. Fr. @Piumal1999  but some years are missing

**Comment by Hasthikagirihagama** [comment]
These are the years of prize giving books we have right now
2008
2009
2011
2012
2013
2014
2015
2016
2018

**Comment by Piumal1999** [comment]
Awesome @Hasthikagirihagama. Is anyone summarizing those content to display on the website, as we discussed in a previous meeting?

**Comment by akeel230** [comment]
Also @Hasthikagirihagama @Piumal1999  @jayasanka-sack we need at least a few pics to display from each of these years' right? 
Shall we ask AMU people to collect pics from year events related to the summary we are making and upload it to a drive maybe?
because pure text without anything to showcase wont be attractive at all.

**Comment by jayasanka-sack** [comment]
Yes, that would be nice! @akeel230 could you please coordinate with @xxxjude and the AMU?

Meanwhile, let's get published whatever the content we have. We can add images when we receive them.
Something is better than nothing ;)
